microservice/main.tf

```diff
@@ -0,0 +1,253 @@
+locals {
+  name_prefix          = "digitalekanaler"
+  shared_config        = nonsensitive(jsondecode(data.aws_ssm_parameter.shared_config.value))
+  service_account_id   = "184465511165"
+  current_region       = data.aws_region.current.name
+  internal_domain_name = "${var.application_name}.${local.shared_config.internal_hosted_zone_name}"
+
+  all_environment_secrets = merge(
+    aws_ssm_parameter.environment_secrets,
+    aws_ssm_parameter.manual_environment_secrets,
+    data.aws_ssm_parameter.external_environment_secrets,
+  )
+}
+
+#########################################
+#                                       #
+# Fargate Task                          #
+#                                       #
+#########################################
+module "task" {
+  source             = "github.com/nsbno/terraform-aws-ecs-service?ref=0.12.1"
+  application_name   = "${local.name_prefix}-${var.application_name}"
+  vpc_id             = local.shared_config.vpc_id
+  private_subnet_ids = local.shared_config.private_subnet_ids
+  cluster_id         = local.shared_config.ecs_cluster_id
+
+  cpu    = 2048
+  memory = 4096
+
+  application_container = {
+    name     = "${local.name_prefix}-${var.application_name}"
+    image    = var.docker_image
+    port     = var.port_number
```

Comment:
Kjører alle tjenestene våre på en standard-port? Jeg er ikke helt enig med meg selv om det burde være en default-verdi eller ikke

Reply:
Oo, er litt usikker jeg også. På den ene siden synes jeg det er digg at vi har gode defaults, så applikasjonene ikke drifter så mye fra hverandre, men på en annen side skjuler man kanskje litt for mye med å ikke sette porten selv 🤷